mobile/src/screens/meditation-home/libs/constants.ts

```diff
@@ -1,13 +1,6 @@
 const mockedData = [
```

Comment:
let's remove this, they shouldn't be mocked it's different pages. And we only need to leave Meditation option

Reply:
let's return this meditation screen by route meditation/list

Reply:
@Liza-Veis i guess i can done it better in this [task ](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/issues/282)related to meditation entries, because for now we can only mock the data and this functionality is built defferently. Later i will connect each page with id of meditation entry, so the mechanism will change. So should i do it in the task linked above ?

Reply:
as i asked on the meeting , we decided to make necessaty changes in next task about journal entries